docs/docsource/Operation-Architecture/Orbit_UI/Bucket_Management/Lifecycle_Management-Transition.rst

```diff
@@ -0,0 +1,47 @@
+Object Lifecycle Management: Transition
+=======================================
+
+**Prerequisite:** You must have established a bucket with at least one object
+in it to transition data from, and another bucket to send transitioned data to.
+
+To establish a lifecycle transition rule:
+
+#. From anywhere in Orbit, click the **WORKFLOWS > Bucket
+   Lifecycle** tab in the left navbar.
+
+   .. image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_select.png
+      :scale: 80%
+
+#. The **Bucket Lifecycle** screen displays.
+
+   |image1|
+
+#. Choose a bucket and pick **Add New Rule > Transition**
+
+   .. image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_add_new_rule.png
+      :scale: 75 %
+      :align: center
+
+#. The **Add New Transition Rule** dialog displays:
+
+   .. image:: ../../Resources/Images/Orbit_Screencaps/Orbit_lifecycle_add_transition_rule.png
+      :scale: 75 %
+      :align: center
+
+   You may name a directory or subdirectory to which the rule applies. Enter
+   a time span after the object's current version was last changed and specify
```

Comment:
~last changed~ last modified

I think people will get the point, but may be worth using "last modified" instead because it is more aligned with the metadata value name which is `LastModified` so may help avoid some questions.

Reply:
Good call